cocos/spine/lib/spine-core.d.ts

```diff
@@ -307,7 +307,7 @@ declare namespace spine {
     }
     class AnimationStateData {
         skeletonData: SkeletonData;
-        animationToMixTime: Map<number>;
+        animationToMixTime: Map<string, number>;
```

Comment:
@bofeng-song Plz review the spine changes.

Reply:
different with https://github.com/EsotericSoftware/spine-runtimes/blob/3.8/spine-ts/build/spine-core.d.ts

Reply:
The definition here is not wrong, but when cocos outputs dts, it does not add the namespace, which causes the map of spine to pollute other code.